SPDX-License-Identifier: MIT

Keep conditional fields within the test-suite and benchmark stanzas.

Index: warp.cabal
--- warp.cabal.orig
+++ warp.cabal
@@ -218,5 +218,5 @@ Test-Suite spec
-  if flag(x509)
-      Build-Depends: crypton-x509
-  if impl(ghc < 8)
-      Build-Depends: semigroups
-                   , transformers
+    if flag(x509)
+        Build-Depends: crypton-x509
+    if impl(ghc < 8)
+        Build-Depends: semigroups
+                     , transformers
@@ -224,12 +224,12 @@ Test-Suite spec
-  if (os(linux) || os(freebsd) || os(darwin)) && flag(allow-sendfilefd)
-      Cpp-Options:   -DSENDFILEFD
-  if os(windows)
-      Cpp-Options:   -DWINDOWS
-      Build-Depends: time
-                 , unix-compat >= 0.2
-  else
-      Build-Depends: unix
-      Other-modules: Network.Wai.Handler.Warp.MultiMap
-  if impl(ghc >= 8)
-      Default-Extensions:  Strict StrictData
-  Default-Language:     Haskell2010
+    if (os(linux) || os(freebsd) || os(darwin)) && flag(allow-sendfilefd)
+        Cpp-Options:   -DSENDFILEFD
+    if os(windows)
+        Cpp-Options:   -DWINDOWS
+        Build-Depends: time
+                   , unix-compat >= 0.2
+    else
+        Build-Depends: unix
+        Other-modules: Network.Wai.Handler.Warp.MultiMap
+    if impl(ghc >= 8)
+        Default-Extensions:  Strict StrictData
+    Default-Language:     Haskell2010
@@ -261,4 +261,4 @@ Benchmark parser
-  if flag(x509)
-      Build-Depends: crypton-x509
-  if impl(ghc < 8)
-      Build-Depends: semigroups
+    if flag(x509)
+        Build-Depends: crypton-x509
+    if impl(ghc < 8)
+        Build-Depends: semigroups
@@ -266,10 +266,10 @@ Benchmark parser
-  if (os(linux) || os(freebsd) || os(darwin)) && flag(allow-sendfilefd)
-    Cpp-Options:   -DSENDFILEFD
-    Build-Depends: unix
-  if os(windows)
-    Cpp-Options:   -DWINDOWS
-    Build-Depends: time
-                 , unix-compat >= 0.2
-  if impl(ghc >= 8)
-      Default-Extensions:  Strict StrictData
-  Default-Language:     Haskell2010
+    if (os(linux) || os(freebsd) || os(darwin)) && flag(allow-sendfilefd)
+      Cpp-Options:   -DSENDFILEFD
+      Build-Depends: unix
+    if os(windows)
+      Cpp-Options:   -DWINDOWS
+      Build-Depends: time
+                   , unix-compat >= 0.2
+    if impl(ghc >= 8)
+        Default-Extensions:  Strict StrictData
+    Default-Language:     Haskell2010
